modules/end-user-guide/examples/proc_using-the-che-prod-editor-file-to-select-an-ide.adoc

```diff
@@ -0,0 +1,17 @@
+[id="using-the-che-prod-editor-file-to-select-an-ide_{context}"]
+== Using the {prod-short} editor file to select an IDE
+
+Use the `che-editor.yaml` file to define a default IDE for the project users. For a list of supported IDs, see xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[Optional parameters for the URLs for starting a new workspace].
+
+.Procedure
+
+* Place the `che-editor.yaml` file in the `.che` folder in the root of your project, and point it at the ID of the selected IDE:
++
+```
+id: che-incubator/che-idea/latest
+```
+
+.Additional resources
+
+* Link to a sample file: https://github.com/che-samples/gradle-demo-project/blob/devfilev2/.che/che-editor.yaml
```

Comment:
Determine whether Che docs are allowed to link to GitHub repos. OpenShift writers are effectively banned from doing so. Instead, they are instructed to integrate the content into the documentation. (I dislike this rule, but must mention it anyway.)